docker-app/Dockerfile

```diff
@@ -85,7 +93,8 @@ ENTRYPOINT ["/entrypoint.sh"]
 FROM base AS webserver_runtime
 ENV LOGGER_SOURCE=app
 COPY ./requirements/requirements_runtime.txt requirements/requirements_runtime.txt
-RUN pip3 install -r requirements/requirements_runtime.txt
+RUN pip3 install -r requirements/requirements_runtime.txt \
+    && rm requirements/requirements_runtime.txt
```

Comment:
What do we win by removing the requirements.txt file?

Reply:
A few bytes of free storage. :)

Reply:
Please sacrifice a few bytes for now :) there are actually a bunch of other files which can safely be removed too, but please don't. Open to revisit this in the future.

Reply:
Done, i also squashed all commits